packages/ag-charts-types/src/chart/toolbarOptions.ts

```diff
@@ -45,28 +45,65 @@ export interface AgToolbarButton {
     value: any;
 }
 
-export type AgIconName = IconNameAnnotation | IconNameZoom;
```

Comment:
These look like breaking changes - were these types used in publicly documented options?

Reply:
Discussed offline, there are two cases that we need to deprecate as they are not in the new icon set:
- `zoom-in-alt`
- `zoom-out-alt`